web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -457,11 +457,11 @@ import {get_string_diff} from "../../src/util";
 
             switch (pseudo_keycode) {
                 case 9: // tab
-                    if (!this.options.tabIsEnter) {
-                        return;
-                    }
                 case 13: // enter
                 case 27: // escape
+                    if (pseudo_keycode === 9 && !this.options.tabIsEnter) {
+                        return;
+                    }
                     e.preventDefault();
                     break;
 
```

Comment:
I think here I'd prefer to just have the `case 9` include its own copy of the `e.preventDefault(); break;` code -- this change feels a lot harder to read.

Reply:
Fair enough - I updated the first two to copy the next block's code, and kept the third as it was